load_data.py

```diff
@@ -97,4 +97,7 @@ def data_loader(dataset, file_name, dtype, use_grayscale=False):
         labels = to_categorical(np.asarray(data_dict['fine_labels'], dtype=dtype),
                 num_classes=100)
 
-    return data, labels
+    if data_length is None:
+        return data, labels
+    else:
+        return data[:data_length], labels[:data_length]
```

Comment:
let's shuffle this and i'll approve it.

Reply:
This is not a good idea, since we want to use it for the comparison of the hyperparameters. They should all be tested on the same subset

Reply:
No you are right, this is important for the distribution of the classes